### DISCUSSION TITLE
jq: error (at <stdin>:1): Cannot iterate over null (null)

### DISCUSSION THREAD

**Discussion by AuZ-ASD** [BlazejosP/huawei-sun2000-API-CLI#37] [Q&A]
Hi,

now I'm one step further.. next error:

> root@munin:/opt/huawei-sun2000-API-CLI-master# ./fusionsolarapp.sh
> 
> Please wait connecting!
> Connection to API
> 
> API login to server OK
> Username & Password accepted by Huawei Server
> Normal Status
> jq: error (at <stdin>:1): Cannot iterate over null (null)
> 
> ndefined Error
> \nReturned data:
> root@munin:/opt/huawei-sun2000-API-CLI-master#

Thanks in advance!

**Comment by Fa6s** [comment]
similar here. got an jq: error (at <stdin>:1): Cannot iterate over null (null)
at the login showng getstationlist

**Comment by BlazejosP** [comment]
Hello I spouse that you have this problem because old login function that is this part of software which need rewriting read this issue and in my opinion that is the reason why. #34
